packages/gatsby-source-wordpress/package.json

```
@@ -1,53 +1,102 @@
+    "glob": "^7.1.6",
+    "got": "^11.7.0",
+    "graphql-query-compress": "^1.2.2",
+    "lodash": "^4.17.19",
+    "node-fetch": "^2.6.0",
+    "p-queue": "^6.3.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "read-chunk": "^3.2.0",
+    "replaceall": "^0.1.6",
+    "semver": "^7.3.2",
+    "valid-url": "^1.0.9"
   },
-  "deprecated": false,
   "devDependencies": {
-    "@babel/cli": "^7.12.1",
-    "@babel/core": "^7.12.3",
-    "babel-preset-gatsby-package": "^0.13.0-next.0",
-    "cross-env": "^7.0.3"
+    "@babel/cli": "^7.6.4",
+    "@babel/core": "^7.6.4",
+    "@babel/plugin-proposal-class-properties": "^7.10.1",
+    "@babel/plugin-proposal-private-methods": "^7.10.1",
+    "@types/semver": "^7.3.4",
+    "babel-jest": "^25.1.0",
+    "babel-plugin-import-globals": "^2.0.0",
+    "babel-plugin-module-resolver": "4.0.0",
+    "babel-preset-gatsby": "0.13.0-next.0",
+    "babel-preset-gatsby-package": "0.13.0-next.0",
+    "cross-env": "^5.2.1",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^25.1.0",
+    "react-test-renderer": "^16.12.0",
+    "rimraf": "^3.0.2",
+    "tree-kill": "^1.2.2",
+    "wait-on": "^4.0.0"
   },
-  "homepage": "https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-source-wordpress#readme",
+  "homepage": "https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-source-wordpress",
   "keywords": [
     "gatsby",
     "gatsby-plugin",
-    "gatsby-source-plugin"
+    "gatsby-plugin-wordpress",
+    "gatsby-source-wordpress",
+    "wordpress"
   ],
   "license": "MIT",
   "peerDependencies": {
-    "gatsby": "^2.0.15"
+    "gatsby": "^2.24.74",
+    "gatsby-plugin-sharp": "^2.6.8",
+    "gatsby-transformer-sharp": "^2.5.10"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/gatsbyjs/gatsby.git",
-    "directory": "packages/gatsby-source-wordpress"
+    "url": "https://github.com/gatsbyjs/gatsby.git"
   },
   "scripts": {
-    "build": "babel src --out-dir . --ignore \"**/__tests__\"",
-    "prepare": "cross-env NODE_ENV=production npm run build",
-    "watch": "babel -w src --out-dir . --ignore \"**/__tests__\""
+    "prepare": "npm run build",
+    "build": "rimraf dist && babel src --out-dir dist --ignore **/__tests__ --extensions \".ts,.js\" --source-maps --root-mode upward && npm run generate-plugin-options-docs",
+    "watch": "npm run build && babel -w src --out-dir dist --ignore **/__tests__ --verbose --extensions \".ts,.js\" --source-maps",
+    "prettier": "prettier . --write",
+    "eslint": "eslint . --ext .ts,.js,.jsx,.tsx",
+    "prettier-check": "prettier . --check",
+    "lint": "yarn prettier-check && yarn eslint",
+    "generate-plugin-options-docs": "node ./generate-plugin-options-docs.js",
+    "docker-start": "docker-compose up --build --force-recreate -V --remove-orphans",
+    "docker-rebuild": "./docker/rebuild.sh"
   },
   "engines": {
-    "node": ">=10.13.0"
+    "node": ">=10.0.0"
```

We support 10.13 nothing lower. This is first LTS
```suggestion
    "node": ">=10.13.0"
```